pkg/models/config.go

```diff
@@ -2,7 +2,7 @@
 package models
 
 type TestSet struct {
-	PreScript  string            `json:"pre_script" bson:"pre_script" yaml:"pre_script"`
-	PostScript string            `json:"post_script" bson:"post_script" yaml:"post_script"`
-	Template   map[string]string `json:"template" bson:"template" yaml:"template"`
+	PreScript  string            `json:"pre_script" bson:"pre_script" yaml:"pre_script" omitempty`
```

Comment:
Correct the config file format:  `json:"pre_script" bson:"pre_script" yaml:"pre_script, omitempty"` can be an example.